common/sgx/collateral.c

```diff
@@ -340,6 +340,28 @@ oe_result_t oe_validate_revocation_list(
         "Failed to parse TCB info or Platform TCB is not up-to-date. %s",
         oe_result_str(result));
 
+    if (memcmp(
+            parsed_extension_info.fmspc,
```

Comment:
So we used pck cert to get `parsed_extension_info` from sgx and used the `fmspc` in that `parsed_extension_info` to retrieve endorsement and later parse the its tcb json to `parsed_tcb_info`. So their `fmspc` should match